typescript-json-schema.ts

```diff
@@ -584,7 +584,11 @@ export class JsonSchemaGenerator {
             }
             if (this.args.generateRequired) {
                 const requiredProps = props.reduce((required: string[], prop: ts.Symbol) => {
-                    if (!(prop.flags & ts.SymbolFlags.Optional) && !(<any>prop).mayBeUndefined) {
+                    if (!(prop.flags & ts.SymbolFlags.Optional)
+                        && !(<any>prop).mayBeUndefined
+                        && prop["tags"].find((x: any): boolean => {
```

Comment:
Can you reuse `otherAnnotations` or just call `parseCommentsIntoDefinition` instead of reading the annotations again?